plugin/sampling/strategystore/static/strategy_store_test.go

```diff
@@ -464,19 +473,64 @@ func TestAutoUpdateStrategyErrors(t *testing.T) {
 	assert.Len(t, logs.FilterMessage("failed to update sampling strategies").All(), 2)
 }
 
-func TestServiceNoPerOperationStrategies(t *testing.T) {
-	store, err := NewStrategyStore(Options{StrategiesFile: "fixtures/service_no_per_operation.json"}, zap.NewNop())
+func TestServiceNoPerOperationStrategies5720ToggleOn(t *testing.T) {
```

Comment:
similar to the main function, I would prefer the test name to be "clean" from bug-fix ticket, and only the to-be-remove test to have a more complex name